lib/bin/decorated.js

```diff
@@ -6,10 +6,25 @@ require('./loadenv');
 var express = require('express');
 var path = require('path');
 var mustacheExpress = require('mustache-express');
+var fs = require('fs');
+
 var getDecorator = require('./decorator');
 
 var server = express();
 
+var renderEnvSettingsFile = function renderEnvSettingsFile(settingsJsonFile, settingsName) {
+  if (settingsJsonFile && settingsName) {
+    var data = JSON.parse(settingsJsonFile);
+    return 'window.' + settingsName + ' = {' + data.map(function (entry) {
+      return entry + ': \'' + process.env[entry] + '\'';
```

Comment:
This is unsafe, the use of `process.env[entry]` is vulnerable to XSS (either by accident or deliberately).

For example if `process.env[entry] = "Hello ' + alert(1) + '` the script will fail. (Or for any occurence of `'` in the variable). This:

```javascript
return JSON.stringify(entry) + ': ' + JSON.stringify(process.env[entry]);`
```

is safer. But still vulnerable to XSS if the code is embedded into HTML (might not be that today, but somebody might find out it's a good idea in the future...), in that case you must [account for this](https://sophiebits.com/2012/08/03/preventing-xss-json.html) as well.